lib/config/fileconf.go

```diff
@@ -468,6 +468,9 @@ type Auth struct {
 	// expires. The empty string implies no message should be sent prior to
 	// disconnection.
 	ClientIdleTimeoutMessage string `yaml:"client_idle_timeout_message,omitempty"`
+
+	// MessageOfTheDay is a banner that a user must acknowledge during a `tsh login`.
+	MessageOfTheDay string `yaml:"message_of_the_day,omitempty"`
```

Comment:
If this will be updated regularly (daily?) should it be a dynamic resource?

Reply:
Even with the MotD being in the config file, an admin can still change it without bringing down the cluster with a `SIGHUP`. I'm not opposed to making a new dynamic resource for it, but the consensus was that a new resource was overkill.

Reply:
Ahh I didn't know about the SIGHUP trick, sounds good